README.md

```diff
@@ -22,12 +22,13 @@ servers and syncing data across devices and applications.
 
 To develop remoteStorage-enabled apps, you need to have a
 remoteStorage-compatible storage account. We recommend
-[reStore](https://github.com/jcoglan/restore) for running a local test server.
```

Comment:
Also good point about the wiki, though. :) I think we should replicate that at least in the beginner's guide, if it isn't there yet.